osu.Game.Rulesets.Osu/Difficulty/OsuPerformanceCalculator.cs

```diff
@@ -181,8 +175,8 @@ private double computeAimValue(ScoreInfo score, OsuDifficultyAttributes attribut
                     estimateImproperlyFollowedDifficultSliders = Math.Min(countSliderEndsDropped + countSliderTickMiss, estimateDifficultSliders);
                 }
 
-                double sliderNerfFactor = (1 - attributes.SliderFactor) * Math.Pow(1 - estimateImproperlyFollowedDifficultSliders / estimateDifficultSliders, 3) + attributes.SliderFactor;
-                aimValue *= sliderNerfFactor;
+                double sliderNerfFactor = (1 - attributes.SliderFactor) * (1 - estimateImproperlyFollowedDifficultSliders / estimateDifficultSliders) + attributes.SliderFactor;
```

Comment:
Considering the known issue with this nerf is the lack of cubing resolved by the line below, I don't think touching the core nerf factor should be in scope of this change.

Reply:
> Considering the known issue with this nerf is the lack of cubing resolved by the line below, I don't think touching the core nerf factor should be in scope of this change.

it's the opposite of what you think
the way I've done this is not touching core nerf factor, when any other variation of this are changing the current curve

https://www.desmos.com/calculator/z2bdvr5iht
Black curve is live, Red line is this PR, Purple line is what you proposed